swan-lake/learn-the-platform/configure-observe/configure-ballerina-programs/provide-values-to-configurable-variables.md

```diff
@@ -69,25 +69,27 @@ the respective types.
-| record[]                 | <code>type  Person  record {</code><br/>    <code>string  name;</code><br/>    <code>int  age;</code><br/><code>};</code><br/>  <code>configurable   Person[]  peope = ?;</code>                                                                                                                      | Array of TOML tables             | `[[people]]`<br/>  `name = "John"`<br/> `age = 45`<br/> `[[people]]`<br/>  `name = "Jack"`<br/> `age = 32`                                  |
-| table                    | <code>configurable   table &lt;map&lt;string&gt;&gt; users = ?; </code>                                                                                                                                                                                                                               | Array of TOML tables             | `[[users]]`<br/> `name = "Tom"`<br/> `occupation = "Software Engineer"`<br/> `[[users]]`<br/> `name = "Harry"`<br/> `occupation = "Doctor"` |
+| Ballerina type           | Ballerina example                                                                                                                                                                                                                                                                                     | TOML type                                                                                                                       | TOML example                                                                                                                                |
+|--------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------|
+| int, byte                | <code>configurable   byte  age = ?;</code><br/>  <code>configurable   int  port = ?;</code>                                                                                                                                                                                                           | Integer                                                                                                                         | `age = 25` <br/>  `port = 9090`                                                                                                             |
```

Comment:
configurable spec has following order in the types.
anydata : =  () | boolean | int | float | decimal | string | xml
             | anydata[] | map<anydata> | table<map<anydata>>

Shall we have same order?